taxi/models.py

```diff
@@ -0,0 +1,35 @@
+from django.db import models
+from django.contrib.auth.models import AbstractUser
+
+from taxi_service import settings
+
+
+class Manufacturer(models.Model):
+    name = models.CharField(max_length=63, unique=True)
+    country = models.CharField(max_length=63)
+
+    def __str__(self) -> str:
+        return self.name
+
+
+class Driver(AbstractUser):
+    license_number = models.CharField(max_length=63, unique=True, null=True)
+
+    class Meta:
+        verbose_name = "driver"
+        verbose_name_plural = "drivers"
+
+
+class Car(models.Model):
+    model = models.CharField(max_length=63)
+    manufacturer = models.ForeignKey(
+        to=Manufacturer, on_delete=models.CASCADE, related_name="cars"
+    )
+
+    drivers = models.ManyToManyField(
+        settings.AUTH_USER_MODEL, related_name="cars"
+    )
+
+    def __str__(self) -> str:
+        return f"{self.model}"  # FIXME <-Here is an error(caused 'model' att).
+        # FIXME Got to use f_String, or rename our attribute"""
```

Comment:
this should work without f-string, can u show an error?

Reply:
if att named "model" - is lighted like a bad annotation
![Screenshot 2023-03-21 at 23 15 54](https://user-images.githubusercontent.com/122725491/226742696-6d9fa21a-d3d0-4364-b308-b5d3e3449e12.png)
but if we add 1 letter - all is fine. IDK whats a problem
![Screenshot 2023-03-21 at 23 16 01](https://user-images.githubusercontent.com/122725491/226742709-a3eafde0-d0a0-455f-8bcf-34118472ab41.png)

Reply:
Just hint. No errors